packages/server-wallet/docs/index.md

```diff
@@ -0,0 +1,183 @@
+# What is a state channel wallet?
+
+A node in a state channel network. See [nitro protocol](https://docs.statechannels.org/`).
+
+More concretely, a state channel wallet _is_:
+
+- a queryable store of keys, states and signatures
+- a run-loop, triggered by
+
+  - blockchain events
+  - API calls
+  - messages from counterparties
+
+  that
+
+  - signs states and transactions
+  - stores them
+  - passes them to external actors
+
+- an event emitter
+
+We usually talk of “The App” as the actor that makes the API calls and binds to the events. The wallet can be a library or run in a separate process to The App (e.g. on a server, or run in an iframe).
+
+A state channel wallet _is not_:
+
+- equipped with a graphical user interface (although this might be a desired plugin)
+
+A state channel wallet has the following _runtime dependencies_:
+
+- a message delivery service (deferred to The App)
+- a blockchain provider (e.g. infura)
+- a database (could be a Postgres server or indexedDB)
+
+The following diagram describes the various components in the architecture. The scope of this document is limited to the components left of the dividing line.
+
+```
+                                                    |
+                     +-----------------+            |     +--------------+
+                     |                 |            |     |              |
+                     |  CHAIN-SERVICE  <------------------>  BLOCKCHAIN  |
```

Comment:
Sadly I don't think this will work because the whole diagram is inside a code fence. 

If only github [supported mermaid diagrams](https://github.community/t/feature-request-support-mermaid-markdown-graph-diagrams-in-md-files/1922/105), which could be a nice solution.